vue/src/vueTextMask.js

```diff
@@ -0,0 +1,51 @@
+import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
+
+/*
+ * Vue Directive - Text Masking with Vanilla Text Mask
+ * https://github.com/text-mask/text-mask/tree/master/vanilla
+ *
+ * This directive binds input masking via Text Mask. Once the element is
+ * destroyed it will be unbinded. Pass an options object, same as you would
+ * normally for this plugin, as a parameter on the element being bound.
+ */
+
+let textMaskInputElement
```

Comment:
I think it's better to move this inside the exported object as `this.textMaskInputElement`